ChangeLog

```diff
@@ -1,3 +1,13 @@
+== 2025-08-05 version 3.0.0
+
+* Remove partial_delete API (#153)
+
+* Support trino type query (#152)
+
+* Clean up test certs (#149)
+
+* Update API responses with new  id, user_id, and description properties for table list, schedule list, result list, database list APIs (#148, #147, #146, #145, #151)
```

Comment:
Sorry, but we actually revert these changes. Kindly remove this line from change log 🙇